docs/resources/bms_instance.md

```diff
@@ -0,0 +1,217 @@
+
+## Argument Reference
+
+-> **NOTE:** If the `user_data` field is specified for a Linux ECS that is created using an image with Cloud-Init
+installed, the `admin_pass` field becomes invalid.
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the instance.
+    If omitted, the provider-level region will be used.
+    Changing this creates a new instance.
+
+* `name` - (Required, String) Specifies a unique name for the instance.
+    The name consists of 1 to 64 characters, including letters, digits, underscores (_), hyphens (-), and periods (.).
+
+* `image_id` - (Required, String, ForceNew) Specifies the image ID of the desired image for the instance.
+    Changing this creates a new instance.
+
+* `flavor_id` - (Required, String, ForceNew) Specifies the flavor ID of the desired flavor for the instance.
+    Changing this creates a new instance.
+
+* `op_svc_userid` - (Required, String, ForceNew) Specifies the user ID. You can obtain the user ID
+    from My Credential on the management console. Changing this creates a new instance.
+
+* `user_data` - (Optional, String, ForceNew) Specifies the user data to be injected during the instance creation.
+    Text and text files can be injected.
+    Changing this creates a new server.
+
+* `admin_pass` - (Optional, String, ForceNew) Specifies the administrative password to assign to the instance.
+    Changing this creates a new instance.
+
+* `key_name` - (Optional, String, ForceNew) Specifies the name of a key pair to put on the instance.
+    The key pair must already be created and associated with the tenant's account.
+    Changing this creates a new instance.
+
+* `security_groups` - (Optional, List, ForceNew) Specifies a array of one or more security group ids to associate
+    with the instance.
+
+* `availability_zone` - (Required, String, ForceNew) Specifies the availability zone in which to create
+    the instance. Please following [reference](https://developer.huaweicloud.com/intl/en-us/endpoint?BMS)
+    for the values. Changing this creates a new instance.
+
+* `vpc_id` - (Required, List, ForceNew) Specifies id of vpc in which to create the instance.
+
+* `nics` - (Required, List, ForceNew) Specifies an array of one or more networks to attach to the instance.
+    The network object structure is documented below.
+    Changing this creates a new instance.
+
+* `eip_id` - (Optional, String, ForceNew) The ID of the EIP. Changing this parameter will create a new resource.
+
+
+-> **Note:** If the eip_id parameter is configured, you do not need to configure the bandwidth parameters:
+  `iptype`, `eip_charge_mode`, `bandwidth_size`, `share_type` and `bandwidth_charge_mode`.
+
+* `iptype` - (Optional, String, ForceNew) Elastic IP type. Changing this parameter will create a new resource.
+
+* `eip_charge_mode` - (Optional, String, ForceNew) Elastic IP billing type. Changing this parameter will create a new resource.
```

Comment:
The suffix name should be `charging_mode`.

Reply:
`charge_mode` is also used in resource cce node, it's better to be consistent